### PR TITLE
Task1: Try python essentials

### PR DESCRIPTION
**Pull request by ipasic-softserve**

### CODE REVIEW

File: lab1/Pasichnichenko_Task1.py [hunk 1]
```diff
@@ -0,0 +1,12 @@
+# 3.1
+print("'Multi-paradigm programming languages':Task 1")
+print("Pasichnichenko Illia IKM-221a")
+
+# 3.2
+for i in range(45):
+    separator = ', ' if i < 44 else ''
```
Comment (vloooo) [inline lab1/Pasichnichenko_Task1.py:7]: Wonderful finding! ternary operator is awesome.

File: lab1/Pasichnichenko_Task1.py [hunk 2]
```diff
@@ -0,0 +1,12 @@
+# 3.1
+print("'Multi-paradigm programming languages':Task 1")
+print("Pasichnichenko Illia IKM-221a")
+
+# 3.2
+for i in range(45):
```
Comment (vloooo) [inline lab1/Pasichnichenko_Task1.py:6]: please look through these links and try to refactor your code:
https://www.w3schools.com/python/python_lists_comprehension.asp
https://www.w3schools.com/python/ref_string_join.asp

File: lab1/Pasichnichenko_Task1.py (hunk 1 above)
Comment (Pasichnichenkoilya) [reply lab1/Pasichnichenko_Task1.py:7]: tnx :)

**Comment by vloooo** [comment]
Why have you merged the PR?) I haven't approve it yet.

File: .idea/vcs.xml [hunk 3]
```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (vloooo) [inline .idea/vcs.xml:1]: please add the file to gitignore:
https://www.atlassian.com/git/tutorials/saving-changes/gitignore

**Pull request merged**